app/src/openxr/cpp/OpenXRLayers.h

```diff
@@ -190,7 +190,7 @@ class OpenXRLayerBase : public OpenXRLayer {
   }
 
 protected:
-  virtual ~OpenXRLayerBase() { Destroy(); }
+  virtual ~OpenXRLayerBase() {}
```

Comment:
I wouldn't recommend this change without knowing why calling the Destroy function causes the issue. If we want to explore an speculative fix like this, I'd suggest to get back the explicit call to the Destroy function that was part of the DeviceDelegateOpenXR::DeleteLayer function's logic before the change you mentioned as culprit of the regression.